ipa-core/src/protocol/ipa_prf/prf_sharding/mod.rs

```diff
@@ -430,8 +438,14 @@ where
         for<'a> TransposeFrom<&'a BitDecomposed<Replicated<Boolean, B>>, Error = LengthError>,
 {
     // Get the validator and context to use for Boolean multiplication operations
-    let binary_validator = sh_ctx.narrow(&Step::BinaryValidator).validator::<Boolean>();
-    let binary_m_ctx = binary_validator.context();
+    // The maximum number of multiplications per user is:
+    // events-1 * (BK bits + TS bits + 4 * TV bits + 3 )
+    let chunk_size = TARGET_PROOF_SIZE
```

Comment:
it is not immediately obvious why 4 * TV + 3, maybe you could elaborate on that here?